src/engine/sync/basesyncablelistener.cpp

```diff
@@ -187,9 +187,9 @@ void BaseSyncableListener::checkUniquePlayingSyncable() {
 
 void BaseSyncableListener::setPlayingSyncable(Syncable* pSource) {
     Syncable *currentSyncable = pSource;
-    // Rearrange the syncables with the playing syncables clustered in the beginning
-    // and appending the current syncable in the end of the playing syncables
-    for (int i=0 ; i<m_syncables.length() ; ++i) {
+    // Sort the syncables with the playing syncables clustered in the beginning
+    // and appending the current syncable at the end of the playing syncables
```

Comment:
Please add a note about the reason for the function here as well: "This allows for keeping track of the first played deck."

Reply:
Updated.